src/ahbichtfunctionsclient/__init__.py

```diff
@@ -37,3 +43,46 @@ async def get_condition_expression(self, package_key: str) -> PackageKeyConditio
                 response_body = await response.json()
                 result = PackageKeyConditionExpressionMappingSchema().load(response_body)
                 return result
+
+
+# pylint: disable=too-few-public-methods
+class HochfrequenzConditionResolver:
+    """
+    A package resolver that uses a REST API (by Hochfrequenz) to request a condition text for a given condition key.
```

Comment:
```suggestion
    Uses a REST API (by Hochfrequenz) to retrieve a condition text for a given condition key.
```
* Es ist kein Package Resolver.
* "retrieve", siehe oben.